smoke-release-testing/mgconsole/nested_indices.bash

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
If you take a look [here](https://github.com/memgraph/mage/blob/main/smoke-release-testing/mgconsole/indices.bash) there is an indices file that can hold different type of indices, Buda started adding it under it's own PRa: https://github.com/memgraph/mage/pull/607/files